src/JoomlaBrowser.php

```diff
@@ -99,6 +99,24 @@ protected function instantiateLocator()
 		$this->locator = new $class;
 	}
 
+	/**
+	 * Locator getter
+	 *
+	 * @param   string  $locator  Locator to get
+	 *
+	 * @return mixed|false
+	 * @since  3.8.11
+	 */
+	public function getLocatorPath($locator)
```

Comment:
could you please change this name to $path?, because `$this->locator->$locator` could be very confusing to the fresh eye.

Reply:
ok there you go ;)